### ISSUE TITLE
Is this library still being supported?

### ISSUE THREAD

**Issue by haizop** [influitive/apartment#591]
Checking in here. I am evaluating this gem for an application and I noticed a few things:
1) No commits since  May 23, 2018
2) No reviews of many open PRs
3) No responses to many open issues by library contributors
4) The Travis build pipeline does not appear to be working properly for new PRs.

@bradrobertson @meganemura Are you all planning on continuing support for this gem? If necessary, will it be updated to support Rails 6?

Thanks for all your great work. I am hoping that this gem will be supported into the future.

**Comment by drosboro** [comment]
This will soon become a big question, when Rails 6 is released in a month or so:

```
apartment (~> 2.2.0) was resolved to 2.2.0, which depends on
      activerecord (< 6.0, >= 3.1.2)
```

I'm not sure if this is a simple matter of changing dependencies or if more work is required - but this could be a major show-stopper for my apps that are currently using apartment / Rails 5.

**Comment by haizop** [comment]
So it seems like the answer is no...

@thedaviddias  - Any plans by Influitive to either maintain this gem or find new maintainers? If not, are you planning to archive this repository?

**Comment by thedaviddias** [comment]
@haizop I'm not currently working on that project but I'm going to ask who is to give you an answer, sorry for not giving you a response sooner.

**Comment by haizop** [comment]
@thedaviddias  - Thanks for your quick response and again thanks to Influitive for this great open source project. I am hoping that it will live on.

**Comment by thedaviddias** [comment]
@haizop Based on the information I was able to gather, we are unfortunately not updating anymore the project. We handed over control to @mikecmpbll, I guess you can ping him. I will keep you posted if I have more information.

**Comment by haizop** [comment]
@mikecmpbll -  Are you going to maintain this gem? If not, are you planning to find new maintainers or archive this repository?

**Comment by mikecmpbll** [comment]
hi folks. sorry for the delay. i picked up some collaborator rights from brad because we're big users of the gem at my company and we needed to get some changes in :). i never had any intentions of being a super active maintainer i'm afraid—i simply don't have the time. influitive have no intentions to maintain the gem because they don't use this solution anymore.

as it happens i'll be doing a big piece of work in this area soon, for work reasons, so i'll likely be very active here during that upcoming period. i'd be more than happy to add additional collaborators though (i think i can!)

**Comment by tfwright** [comment]
I would be able to spend some time to collaborate on this.

**Comment by porterbayne** [comment]
thanks @mikecmpbll .  let us know how we can help.

**Comment by tachyons** [comment]
I've fixed 4th point in #603

**Comment by mikecmpbll** [comment]
i'll be putting some hours in over the next few weeks, so i'll close this for now but feel free to continue the discussion if useful.
